### DISCUSSION TITLE
The AirBand AM transmission is pretty bad

### DISCUSSION THREAD

**Discussion by evangel76** [egzumer/uv-k5-firmware-custom#247] [General]
Is there something we can do to clean it up, I have recompiled the code and I am willing to code if something is needed to clean it up.

Probably a naive question but it does not hurt to volunteer if a task is possible.

Francois

**Comment by egzumer** [comment]
There is no transmission support. You probably meant AM receiving, it is excellent as for this radio (and I'm not saying about my achievement, I didn't invent the AM-fix), mind you this is FM transceiver, and it is dirt cheap.

You can look at https://github.com/egzumer/uv-k5-firmware-custom/wiki/Development-notes#1o11-am-fix-issue-with-strong-signals
It might give you some idea on how it works, there is also reference manual attached.

This is the central part in code for AM reception:
https://github.com/egzumer/uv-k5-firmware-custom/blob/d0ae34f9b498e7d81d6f2e1a024d91b0215d06ac/radio.c#L893

Feel free to experiment. Waiting to hear about some new discoveries.

**Reply by grimmaceil** [reply]
Is AM transmit worthwhile, especially in the Airband which is out of the filtering range of 135 - 174? Has anyone tried and dine a spectrum analysis for spatter and harmonic interference in the airband? I'm asking because my father is a Ham technician and a sports pilot. I realize this is a cheap radio not designed for this use case. Similarly, I assume it would not be a good idea to transmit on any frequency outside the filtering design such as 1.25 meters. I think it's just great to be able to listen to such a broad range.

**Reply by egzumer** [reply]
You can't transmit in AM with this radio. 
You can search there for spectrum analysis
https://github.com/ludwich66/Quansheng_UV-K5_Wiki/wiki/Measurements
